Splunk Phantom/DomainTools_Guided_Pivots/README.md

```diff
@@ -1,9 +1,14 @@
-## DomainTools Guided Pivots Playbook
+## DomainTools Iris Malicious Tags Playbook
+
+Retrieve the Iris Investigate profile of a domain and automatically flag domains that have been tagged as malicious in DomainTools Iris. The list of malicious tags are configurable by the analyst and can be kept up to date for automation.
+
+#### Needed Asset in Phantom
+The DomainTools playbooks in this repo will require an asset called `domaintoolsiriscreds` with DomainTools API username and key. It is needed to retrieve DomainTools data via the DomainTools API within the playbook.
+<br />
 
-This playbook retrieves the Iris Investigate profile of domain and automatically identifies potential connected infrastructure related to artifacts based on DomainTools Guided Pivot value. The pivot value is set based off of a configurable custom list.
 
 #### Before Installing
-In this directory there is a csv called domaintools_guided_pivot.csv. You will need to copy the contents of this into a custom list on your instance with the same name. The names in the first column are used by custom code on the decision blocks in the PB. The second column are the pivot values that you can adjust to what works best for your needs.
+With the  `domaintoolsiriscreds` asset loaded in Phantom, create a custom list named `iris_malicious_tags`.  In this directory there is a csv called iris_malicious_tags.csv. Copy the contents of this into the custom list. The values in the list can be changed to any tags you wish to have flagged.
 
 #### Installation 
-After creating the custom list mentioned above, download the tar file in this directory and import the PB using that file. The assets in PB are for your DomainTools credentials. Point the asset to your dt credentials asset, save the PB and give it a shot.
\ No newline at end of file
+Download the tar file in this directory and import the playbook using that file. The assets in playbook are for your DomainTools credentials. Point the asset to your DomainTools `domaintoolsiriscreds` asset, save the playbook, and give it a shot.
```

Comment:
Are these changes intended? The playbook in questions _appears_ to be our `DomainTools_Guided_Pivots`, but the content seems to align with our Malicious Tags Playbook.